### PR TITLE
Rename 

### LINKED ISSUE #27: Rename key from master to main
### What's your idea?

Rename key from master to main

### Impact

Low

### Urgency

Now

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by giulio-giunta**
## Description
Rename key from master to main

## Related Issue
"resolves #27" 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](https://github.com/GeekZoneHQ/web/blob/main/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: aws/k8s/sealed-secrets/main-key-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
...SmJ1cVVLK3NiRmN4dmlQa21qQkhqSVFwMXV1c1YySlQ5SXdqMmM5cDZVU1ZlNCsKY28rWXBRSURBUUFCbzBJd1FEQU9CZ05WSFE4QkFmOEVCQU1DQUFFd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZApCZ05WSFE0RUZnUVVYVlhhZmJPK21xbm9qWndOYURpZHhWQjBIcmt3RFFZSktvWklodmNOQVFFTEJRQURnZ0lCCkFMSjZjSy9Uc3RtUVZSbkp2Y29HNUZQTGh4SXBqQkZheUFzc3c5MVdHdU9nTVhmVTJmQWNBcnNRY3Y3YzIvQUwKcVkwVFJlaklrdTFLQzlvcHRzdnV4QmFIMnEra3gydFFxSUltR0J0RlhZZmd0bVI3Mi8ybFNwZWUzTFZTODR2RQpOQmhhZHhvOEtRVmJiNkh0TE82Qml0ODFaZitYNnBxaWRNbklqeGFuUkJZZ2JGOVZ4Sm1EczlzenZtMDgwUjhyCkRtbWorU2NaalZ6cGtJaGpJeW5QNGU0UFA4YTd1dGR6bUNEcE5odWx3blNUN21SSk5mcitwV3FmRmRaaFBVSWwKbExlUWN3aEtmMllYbGpLeUtmUHNqWEhZblNKc1ZabVVwZ0ZUeWtzQVYzU0w5UzFIQWNZNUFWa0o1alc3MmU2QQprcVBUS1dESGZhU3BMOFNEQmh3UmhLNnhSN3l2ZHBmc0N1OFQzTmJaOG9lSWwzVk5HMitLTGNLQnlsRlZiSFFzCnhyWUFZSTRHa3Q1Mjk0cTdpTGo3OWNBN3RnL2N2NzJRY09TYlJGTTV5RFlweHNDOGVkdlZTRmFyd1hQNDNmSDMKTmNkaFpUK0FvQ0FQNHNJOHpJWks0SWpNaXRud2FremZsZHl6RU1OVm9sL3lwQlgrM01IRHcxOWdNSkdUNjNUTgp4S28zZW5FeWNoT25WU1c0OWwzaDlhQm9lMEhtRXQ3OVNZM0ZmYlVFM2hXVDBEZDQyb0xrTThuYTh6WU52R2JDClYxdXpZV2FGMk9qM1hmbmFqcTFZZytSdiswd1h0akZtZkxsUytQM09HYm50SERDdlpXYmtOYWVHUkFYcnlmL08KZUowQm1wMXBXMDgrQk56M21kZHZnS0dmMkdITjFZOThoNXcxTlNRbGw3azkKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```
Comment (Freshrojek) [inline aws/k8s/sealed-secrets/main-key-secret.yaml:3]: out of curiosity what does `tls.crt` contain?

File: azure/k8s/sealed-secrets/main-key-secret.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
...SmJ1cVVLK3NiRmN4dmlQa21qQkhqSVFwMXV1c1YySlQ5SXdqMmM5cDZVU1ZlNCsKY28rWXBRSURBUUFCbzBJd1FEQU9CZ05WSFE4QkFmOEVCQU1DQUFFd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZApCZ05WSFE0RUZnUVVYVlhhZmJPK21xbm9qWndOYURpZHhWQjBIcmt3RFFZSktvWklodmNOQVFFTEJRQURnZ0lCCkFMSjZjSy9Uc3RtUVZSbkp2Y29HNUZQTGh4SXBqQkZheUFzc3c5MVdHdU9nTVhmVTJmQWNBcnNRY3Y3YzIvQUwKcVkwVFJlaklrdTFLQzlvcHRzdnV4QmFIMnEra3gydFFxSUltR0J0RlhZZmd0bVI3Mi8ybFNwZWUzTFZTODR2RQpOQmhhZHhvOEtRVmJiNkh0TE82Qml0ODFaZitYNnBxaWRNbklqeGFuUkJZZ2JGOVZ4Sm1EczlzenZtMDgwUjhyCkRtbWorU2NaalZ6cGtJaGpJeW5QNGU0UFA4YTd1dGR6bUNEcE5odWx3blNUN21SSk5mcitwV3FmRmRaaFBVSWwKbExlUWN3aEtmMllYbGpLeUtmUHNqWEhZblNKc1ZabVVwZ0ZUeWtzQVYzU0w5UzFIQWNZNUFWa0o1alc3MmU2QQprcVBUS1dESGZhU3BMOFNEQmh3UmhLNnhSN3l2ZHBmc0N1OFQzTmJaOG9lSWwzVk5HMitLTGNLQnlsRlZiSFFzCnhyWUFZSTRHa3Q1Mjk0cTdpTGo3OWNBN3RnL2N2NzJRY09TYlJGTTV5RFlweHNDOGVkdlZTRmFyd1hQNDNmSDMKTmNkaFpUK0FvQ0FQNHNJOHpJWks0SWpNaXRud2FremZsZHl6RU1OVm9sL3lwQlgrM01IRHcxOWdNSkdUNjNUTgp4S28zZW5FeWNoT25WU1c0OWwzaDlhQm9lMEhtRXQ3OVNZM0ZmYlVFM2hXVDBEZDQyb0xrTThuYTh6WU52R2JDClYxdXpZV2FGMk9qM1hmbmFqcTFZZytSdiswd1h0akZtZkxsUytQM09HYm50SERDdlpXYmtOYWVHUkFYcnlmL08KZUowQm1wMXBXMDgrQk56M21kZHZnS0dmMkdITjFZOThoNXcxTlNRbGw3azkKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```
Comment (Freshrojek) [inline azure/k8s/sealed-secrets/main-key-secret.yaml:3]: the same question here 🤔

File: aws/k8s/sealed-secrets/main-key-secret.yaml (hunk 1 above)
Comment (giulio-giunta) [reply aws/k8s/sealed-secrets/main-key-secret.yaml:3]: It contains the public key to encrypt secrets. The public key can be published, as it cannot decrypt. The private key, on the contrary is not published. 
The public key can be used to encrypt secrets off-line.

File: azure/k8s/sealed-secrets/main-key-secret.yaml (hunk 2 above)
Comment (giulio-giunta) [reply azure/k8s/sealed-secrets/main-key-secret.yaml:3]: Same as above. I added a further change to this PR linked with a configuration problem that prevented the django app to connect to the database

**Review by Freshrojek — APPROVED** [review]

**Pull request merged**